modules/json/src/main/scala/gem/json.scala

```diff
@@ -20,7 +20,10 @@ package object json {
 
   // Angle mapping to signed arcseconds. NOT implicit.
   val AngleMetaAsSignedArcseconds: CodecJson[Angle] =
-    CodecJson.derived[Double].xmap(Angle.fromArcsecs)(_.toSignedDegrees * 3600)
+    CodecJson.derived[BigDecimal]
```

Comment:
Yeah the wire value will be in signed arcseconds. This is for offsets so the values are like `-0.25` or whatever. I think probably it makes sense to pass them as full-precision angles but we're not there yet. This is mostly just a sanity check that we're not doing anything to preclude JSON encoding.